assert_test.go

```diff
@@ -0,0 +1,55 @@
+package gosnowflake
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+)
+
+func assertNil(t *testing.T, actual interface{}, descriptions ...string) {
```

Comment:
Can we replace `interface{}` with `any`?

Reply:
done